### DISCUSSION TITLE
What is local mode supposed to do?

### DISCUSSION THREAD

**Discussion by zdhernandez** [hkchengrex/MiVOS#40] [General]
@hkchengrex  What is local mode supposed to do?
If we press the ctrl key and then click the mouse button the GUI enters in this local mode that I don't understand what is used for, then there is the button finish local to resume the normal behavior. This is in:

```
in interactive_gui line 257 there are
# Local mode related states
self.ctrl_key = False
[self.in](http://self.in/)_local_mode = False
self.local_bb = None
self.local_interactions = {}
self.this_local_interactions = []
self.local_interaction = Nonein interactive_gui line 257 there are
# Local mode related states
self.ctrl_key = False
[self.in](http://self.in/)_local_mode = False
self.local_bb = None
self.local_interactions = {}
self.this_local_interactions = []
self.local_interaction = None
```
![local_control](https://user-images.githubusercontent.com/8461519/176199050-15050d12-268e-454a-9b68-0d811ccec647.jpg)

**Comment by hkchengrex** [comment]
P.4 of the paper describes this. It zooms into a region for fine-processing without affecting the rest of the image. The user is supposed to drag-select a region. In this case you might have selected a single pixel.
